### ISSUE TITLE
Keeping old assets

### ISSUE THREAD

**Issue by jbonhag** [dschmidt/ember-cli-deploy-cp#3]
Thanks for writing this plugin -- it's nice to have such a simple way to deploy.

We're trying to get a "Lightning-esqe" deploy going.  I want to keep assets in a local `html` directory rather than in an S3 bucket, but we lose versioning because it looks like the ember-cli-deploy-cp plugin cleans out old assets as part of the deploy.

Would it be possible to add an option to keep whatever's in the assets directory?

**Comment by dschmidt** [comment]
I actually meant to implement it like that. Will investigate.

**Comment by dschmidt** [comment]
3 possibilities here:
1) bug in `cpr`
2) i'm using it wrong
3) it's not meant to do what we need here

for further reference: https://github.com/davglass/cpr/issues/18

**Comment by jbonhag** [comment]
Yeah, it looks like overwrite is all-or-nothing.  If the directory exists, `overwrite: false` won't do anything, and `overwrite: true` will replace everything.

**Comment by dschmidt** [comment]
Exactly. Yeah, let's wait for a response on the cpr ticket, if it can't be solved with that lib, I'll look for a different one. 

TL;DR: keeping old assets was totally intended when I wrote this plugin, so I will make sure it will behave this way.

**Comment by jbonhag** [comment]
Awesome, thanks @dschmidt!

**Comment by dschmidt** [comment]
Fixed on cpr master, which will become 1.0.0 now.

When that is published I will update and merge the following:
https://github.com/dschmidt/ember-cli-deploy-cp/pull/4

Appreciated if you confirm it works for you now too.

**Comment by dschmidt** [comment]
Released and published ember-cli-deploy-cp@0.3.0

**Comment by jbonhag** [comment]
Works like a dream.  Thanks for taking care of this so quickly. :trophy:
